### DISCUSSION TITLE
Location of generated source files with MVVM toolkit

### DISCUSSION THREAD

**Discussion by DaveMcD56** [CommunityToolkit/dotnet#638] [Q&A] [answered]
Our build pipeline has a Guardian Dotnet Analyzer tool that appears to have problems locating where MVVM tookit source generator is putting the generated source code files.
On local builds I see the generated files going to my local temp folder: C:\Users\<userName>\AppData\Local\Temp\VSGeneratedDocuments\<guid>
Is there any way to control where the generator puts its generated output files?

**Comment by Sergio0694** [comment, marked as answer]
See [these docs](https://learn.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/source-generators-overview#get-started-with-source-generators). You'll want to use `<EmitCompilerGeneratedFiles>` and `<CompilerGeneratedFilesOutputPath>` 🙂

**Reply by DaveMcD56** [reply]
Thank you!
